resources/measures/upgrade_hvac_doas_hp_minisplits/measure.rb

```diff
@@ -1341,24 +1341,30 @@ def run(model, runner, user_arguments)
 
       # add new coils to unitary system object
       unitary_sys.setHeatingCoil(new_dx_heating_coil)
-      unitary_sys.setCoolingCoil(new_dx_cooling_coil) 
+      unitary_sys.setCoolingCoil(new_dx_cooling_coil)
       # set backup heat to meet design heating load
       supp_htg_coil = unitary_sys.supplementalHeatingCoil.get.to_CoilHeatingElectric.get
       supp_htg_coil.setNominalCapacity(dsn_htg_load)
       supp_htg_coil.setEfficiency(1)
 
       # set other features
       unitary_sys.setDXHeatingCoilSizingRatio(1+performance_oversizing_factor)
-      # set cooling design flow rate
-      unitary_sys.setSupplyAirFlowRateMethodDuringCoolingOperation('SupplyAirFlowRate')
-      unitary_sys.setSupplyAirFlowRateDuringCoolingOperation(airflow_stage4)
-      # set heating design flow rate
-      unitary_sys.setSupplyAirFlowRateMethodDuringHeatingOperation('SupplyAirFlowRate')
-      unitary_sys.setSupplyAirFlowRateDuringHeatingOperation(airflow_stage4)
-      # set no load design flow rate
-      unitary_sys.setSupplyAirFlowRateMethodWhenNoCoolingorHeatingisRequired('SupplyAirFlowRate')
-      unitary_sys.setSupplyAirFlowRateWhenNoCoolingorHeatingisRequired(airflow_stage1)
-      
+
+      if model.version < OpenStudio::VersionString.new('3.7.0')
+        # set cooling design flow rate
+        unitary_sys.setSupplyAirFlowRateMethodDuringCoolingOperation('SupplyAirFlowRate')
+        unitary_sys.setSupplyAirFlowRateDuringCoolingOperation(airflow_stage4)
+        # set heating design flow rate
+        unitary_sys.setSupplyAirFlowRateMethodDuringHeatingOperation('SupplyAirFlowRate')
+        unitary_sys.setSupplyAirFlowRateDuringHeatingOperation(airflow_stage4)
+        # set no load design flow rate
+        unitary_sys.setSupplyAirFlowRateMethodWhenNoCoolingorHeatingisRequired('SupplyAirFlowRate')
+        unitary_sys.setSupplyAirFlowRateWhenNoCoolingorHeatingisRequired(airflow_stage1)
+      else
+        unitary_sys.autosizeSupplyAirFlowRateDuringCoolingOperation
+        unitary_sys.autosizeSupplyAirFlowRateDuringHeatingOperation
+        unitary_sys.autosizeSupplyAirFlowRateWhenNoCoolingorHeatingisRequired
```

Comment:
Again this changes measure behavior. Keep the flow rate assignments, don't autosize

Reply:
Oh my bad. Updated to still set the flow rates but with the new method